src/typings/analysisModule.tsx

```diff
@@ -0,0 +1,9 @@
+export interface AnalysisElement {
```

Comment:
Create a common interface name Entity;
```
interface Entity = {...}
export type AnalysisElement = Entity;
export type PillarAnalysisElement = Entity;
```

Reply:
This is temporary and new things will be added